pkg/virt-api/webhooks/validating-webhook/admitters/validate-k8s-utils.go

```diff
@@ -53,6 +55,25 @@ var nodeFieldSelectorValidators = map[string]func(string, bool) []string{
 	metav1.ObjectNameField: ValidateNodeName,
 }
 
+// validateFieldValueNotConformingToDNSLabelRules is function that validate value and checks if given affinities are valid
+func validateFieldValueNotConformingToDNSLabelRules(field *field.Path, value, valueField string) (causes []metav1.StatusCause) {
```

Comment:
Can I ask you why you introduced the `valueField` parameter? IMHO it can create confusion.
Also, in this case the `field` parameter is just a parent `Path`.
Are there any reasons that I am not seeing?
If the problem is the caller param, it is enough to declare the `field` as `k8sfield.NewPath("spec", "hostname")`
Thanks